samples/boards/esp32/spiram_test/boards/yd_esp32.conf

```diff
@@ -0,0 +1,2 @@
+CONFIG_ESP_HEAP_MEM_POOL_REGION_1_SIZE=100000
```

Comment:
Does it have spiram in this ESP32 module version? If not, you can remove this.

Reply:
Done. Thank you.

---

The board is supplied[^1] with one of the following modules:

- ESP32-WROOM-32E-N4
- ESP32-WROOM-32E-N8
- ESP32-WROOM-32E-N16

None of them have SPIRAM/ PSRAM according to the datasheet[^2]. 

Running the [Espressif ESP32 SPIRAM test](https://docs.zephyrproject.org/latest/samples/boards/esp32/spiram_test/README.html) will result in the following error:

![y_spiram_test_error](https://github.com/zephyrproject-rtos/zephyr/assets/103332557/921d305f-8d1d-4c14-8f32-de674e226e2a)

[^1]: Source: VCC-GND® Studio's [product page](https://www.aliexpress.com/item/1005004363550813.html) on AliExpress. Last checked date: Jul 6, 2023.
[^2]: Source: [esp32-wroom-32e_esp32-wroom-32ue_datasheet_en.pdf](https://github.com/zephyrproject-rtos/zephyr/files/11971690/esp32-wroom-32e_esp32-wroom-32ue_datasheet_en.pdf), v1.6.